assets/parquet/config.yaml

```
@@ -0,0 +1,131 @@
+    "array_position(string_columns, 'ctx') > 0 
+    AND element_at(
+      string_values, 
+      array_position(string_columns, 'ctx')
+    ) = 'FlowControlRefresher' 
+    AND array_position(int_columns, 'attr.numTrimmed') > 0 
+    AND element_at(
+      int_values, 
+      array_position(int_columns, 'attr.numTrimmed')
+    ) = 0"
+  # - '"ctx: FlowControlRefresher AND attr.numTrimmed: 0"'
+
```

YAMLlint (1.37.1)

[error] too many blank lines (1 > 0)

(empty-lines)